Fix navigation when `base` path is set and validate that option's value (#1666)
sveltejs/kit: .changeset/soft-bikes-bake.md

---
'@sveltejs/kit': patch
---

> Fix redirect URLs when removing/adding trailing slash when `base` path is set

Reviewer: ```suggestion
Fix navigation when `base` path is set and validate that option's value
```
